optimum/intel/ipex/modeling_base.py

```diff
@@ -231,24 +233,27 @@ def to(self, device: Union[torch.device, str]):
     def can_generate(self):
         return isinstance(self, GenerationMixin)
 
-    def maybe_apply_torch_compile(self):
+    def can_compile(self):
```

Comment:
Please make this method private, let's not add public interface methods that are removed in a couple PRs.

Reply:
fixed